packages/core/src/Ponder.ts

```diff
@@ -174,24 +174,6 @@ export class Ponder {
 
     this.registerServiceDependencies();
 
-    // If any of the provided networks do not have a valid RPC url,
-    // kill the app here. This happens here rather than in the constructor because
-    // `ponder codegen` should still be able to if an RPC url is missing. In fact,
-    // that is part of the happy path for `create-ponder`.
-    const networksMissingRpcUrl: Network[] = [];
```

Comment:
This is unnecessary since the `viem` transport will default to using a public RPC if not specified.